src/components/navigation/Footer/NewsFeed.tsx

```diff
@@ -7,23 +7,38 @@ import './NewsFeed.scss'
 const parser = new Parser()
 
 const News: React.FC = (): ReactElement => {
-  const feed: Item[] = []
-  const [items, setItems] = useState(feed)
+  // Component state to hold new items, error, and loading state.
+  const [items, setItems] = useState<Item[]>([])
+  const [error, setError] = useState<string | null>(null)
+  const [isLoading, setIsLoading] = useState(true)
 
+  // On component mount, fetch and set the news feed, ensuring to handle errors.
   useEffect(() => {
     async function fetchAndSetFeed(): Promise<void> {
-      const results = await parser.parseURL('https://neonewstoday.com/feed/')
-      if (results) {
-        results.items && setItems(results.items)
+      try {
+        setIsLoading(true)
+        setError(null)
+        const results = await parser.parseURL('https://neonewstoday.com/feed/')
+        if (results?.items?.length) {
+          setItems(results.items)
+        } else {
+          setError('No news items found from NNT')
```

Comment:
Is the not found state an error state?

Reply:
Yep! The feed should always have items in it.